### ISSUE TITLE
augtool get output is difficult to use from shell scripts

### ISSUE THREAD

**Issue by kunkku** [hercules-team/augeas#419]
The output of "augtool get <PATH>" starts with "<PATH> =", making it unnecessarily complicated to parse e.g. in a shell script. The path pattern is not even expanded to the actual path, so printing it does not add any value.

I would like to have the path prefix removed from the output by default, or at least a command line option to disable it.

**Comment by lutter** [comment]
I agree that this would be very useful. I'd be almost inclined to change the default behavior of 'augtool get' but I am concerned that we'd break other people's scripts.

If we go with another command line flag (say `--terse`), we'd also need to define the output from other commands in that mode, and make sure it's script friendly. For example, `get` doesn't report an error if you do something crazy like `augtool get /files//*` - it just prints nothing. Similarly, `set` prints `Saved N file(s)` when it saves a change. I'd imagine that for scripts, printing the files that were actually changed would be more useful. Haven't had time to look through other commands ...
